analyses/independent-samples/01-generate-independent-specimens.R

```diff
@@ -100,27 +100,27 @@ tumor_samples <- histology_df %>%
 wgs_samples <- tumor_samples %>%
   dplyr::filter(experimental_strategy == "WGS")
 
-wgs_primary_each <- independent_samples(wgs_samples, tumor_types = "primary", independent_level = "each-cohort", seed = 2020)
-wgs_relapse_each <- independent_samples(wgs_samples, tumor_types = "relapse", independent_level = "each-cohort", seed = 2020)
-wgs_primary_plus_each <- independent_samples(wgs_samples, tumor_types = "prefer_primary", independent_level = "each-cohort", seed = 2020)
+wgs_primary_each <- independent_samples(wgs_samples, tumor_types = "primary", independent_level = "each-cohort", seed = 2020) %>% dplyr::arrange(Kids_First_Participant_ID)
+wgs_relapse_each <- independent_samples(wgs_samples, tumor_types = "relapse", independent_level = "each-cohort", seed = 2020) %>% dplyr::arrange(Kids_First_Participant_ID)
+wgs_primary_plus_each <- independent_samples(wgs_samples, tumor_types = "prefer_primary", independent_level = "each-cohort", seed = 2020) %>% dplyr::arrange(Kids_First_Participant_ID)
 
-wgs_primary_all <- independent_samples(wgs_samples, tumor_types = "primary", independent_level = "all-cohorts", seed = 2020)
-wgs_relapse_all <- independent_samples(wgs_samples, tumor_types = "relapse", independent_level = "all-cohorts", seed = 2020)
-wgs_primary_plus_all <- independent_samples(wgs_samples, tumor_types = "prefer_primary", independent_level = "all-cohorts", seed = 2020)
+wgs_primary_all <- independent_samples(wgs_samples, tumor_types = "primary", independent_level = "all-cohorts", seed = 2020) %>% dplyr::arrange(Kids_First_Participant_ID)
+wgs_relapse_all <- independent_samples(wgs_samples, tumor_types = "relapse", independent_level = "all-cohorts", seed = 2020) %>% dplyr::arrange(Kids_First_Participant_ID)
+wgs_primary_plus_all <- independent_samples(wgs_samples, tumor_types = "prefer_primary", independent_level = "all-cohorts", seed = 2020) %>% dplyr::arrange(Kids_First_Participant_ID)
```

Comment:
I don't think these arranges are necessary - it is really for just writing out the files